src/plugins/workspace/public/workspace_client.ts

```diff
@@ -319,21 +320,21 @@ export class WorkspaceClient implements IWorkspaceClient {
    * @param {Array<{ id: string; type: string }>} objects
    * @param {string} targetWorkspace
    * @param {boolean} includeReferencesDeep
-   * @returns {Promise<IResponse<any>>} result for this operation
+   * @returns {Promise<SavedObjectsImportResponse>} result for this operation
    */
   public async copy(
     objects: Array<{ id: string; type: string }>,
     targetWorkspace: string,
     includeReferencesDeep: boolean = true
-  ): Promise<IResponse<any>> {
+  ): Promise<SavedObjectsImportResponse> {
     const path = this.getPath('_duplicate_saved_objects');
     const body = {
       objects,
       targetWorkspace,
       includeReferencesDeep,
     };
 
-    const result = await this.safeFetch(path, {
+    const result = await this.http.fetch<SavedObjectsImportResponse>(path, {
```

Comment:
is there a reason changed to `http.fetch`?

Reply:
The structure of `SavedObjectsImportResponse` is 
```
export interface SavedObjectsImportResponse {
  success: boolean;
  successCount: number;
  successResults?: SavedObjectsImportSuccess[];
  errors?: SavedObjectsImportError[];
}
```
which is incompatible with `IResponse`